x-pack/plugins/graph/public/plugin.ts

```diff
@@ -150,10 +155,6 @@ export class GraphPlugin
           : AppNavLinkStatus.hidden,
         tooltip: licenseInformation.showAppLink ? licenseInformation.message : undefined,
       }));
-
-      if (home && !licenseInformation.enableAppLink) {
-        home.featureCatalogue.removeFeature('graph');
```

Comment:
There was no guarantee that the home app hadn't already called `featureCatalogue.get()` before this statement could run.